google/cloud/pubsublite/internal/partition_publisher.h

```diff
@@ -96,9 +100,9 @@ class PartitionPublisher
   std::deque<std::deque<MessageWithPromise>>
       unsent_batches_;  // ABSL_GUARDED_BY(mu_)
   std::deque<std::deque<MessageWithPromise>>
-
       in_flight_batches_;  // ABSL_GUARDED_BY(mu_)
   bool writing_ = false;   // ABSL_GUARDED_BY(mu_)
+  absl::optional<promise<void>> reading_;  // ABSL_GUARDED_BY(mu_)
```

Comment:
The robots would like you to reformat this code:

```
diff --git a/google/cloud/pubsublite/internal/partition_publisher.h b/google/cloud/pubsublite/internal/partition_publisher.h
index a750951..961caf0 100644
--- a/google/cloud/pubsublite/internal/partition_publisher.h
+++ b/google/cloud/pubsublite/internal/partition_publisher.h
@@ -100,8 +100,8 @@ class PartitionPublisher
   std::deque<std::deque<MessageWithPromise>>
       unsent_batches_;  // ABSL_GUARDED_BY(mu_)
   std::deque<std::deque<MessageWithPromise>>
-      in_flight_batches_;  // ABSL_GUARDED_BY(mu_)
-  bool writing_ = false;   // ABSL_GUARDED_BY(mu_)
+      in_flight_batches_;                  // ABSL_GUARDED_BY(mu_)
+  bool writing_ = false;                   // ABSL_GUARDED_BY(mu_)
   absl::optional<promise<void>> reading_;  // ABSL_GUARDED_BY(mu_)
 
   std::unique_ptr<AlarmRegistry::CancelToken> cancel_token_;
```

* Raw: https://storage.googleapis.com/cloud-cpp-community-publiclogs/logs/google-cloud-cpp/10222/3fa3d30ab2912de7f5cd8ef7a65400013805f429/checkers-checkers/log-62d838c2-28df-4bed-8c67-fdd20038f626.txt